examples/react-todo/src/main.tsx

```diff
@@ -38,7 +39,11 @@ window.repo = repo
 ReactDOM.createRoot(document.getElementById("root") as HTMLElement).render(
   <RepoContext.Provider value={repo}>
     <React.StrictMode>
-      <App url={docUrl} />
+      <ErrorBoundary fallback={<div>Something went wrong</div>}>
```

Comment:
this officialish error boundary is chatty in the logs. i should silence it and/or replace it with one that is better for tests.